sha2/Cargo.toml

```diff
@@ -10,5 +10,15 @@ keywords = ["crypto", "sha2", "asm"]
 categories = ["cryptography", "no-std"]
 edition = "2018"
 
+[dependencies]
+cpufeatures = "0.2.1"
+
 [build-dependencies]
 cc = "1.0"
+
+[dev-dependencies]
+criterion = {version= "0.3.5",features=["html_reports"] }
```

Comment:
`criterion` is a quite sizable dependency, which makes CI jobs significantly longer. This is why we prefer to keep criterion-based benchmarks in a separate folder (e.g. see the [benches folder](https://github.com/RustCrypto/stream-ciphers/tree/master/benches) in the stream ciphers repo). So I think it could be worth to keep the old benchmarks and add criterion-based ones as a separate crate.